grpc/src/test/java/com/linecorp/armeria/it/grpc/GrpcHttpJsonTranscodingServiceAnnotatedAuthServiceTest.java

```diff
@@ -0,0 +1,154 @@
+import java.lang.annotation.Target;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.armeria.client.BlockingWebClient;
+import com.linecorp.armeria.client.grpc.GrpcClients;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.RequestHeaders;
+import com.linecorp.armeria.common.grpc.GrpcSerializationFormats;
+import com.linecorp.armeria.common.util.UnmodifiableFuture;
+import com.linecorp.armeria.grpc.testing.HttpJsonTranscodingTestServiceGrpc.HttpJsonTranscodingTestServiceBlockingStub;
+import com.linecorp.armeria.grpc.testing.HttpJsonTranscodingTestServiceGrpc.HttpJsonTranscodingTestServiceImplBase;
+import com.linecorp.armeria.grpc.testing.Transcoding;
+import com.linecorp.armeria.internal.common.JacksonUtil;
+import com.linecorp.armeria.server.HttpService;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.annotation.DecoratorFactory;
+import com.linecorp.armeria.server.annotation.DecoratorFactoryFunction;
+import com.linecorp.armeria.server.auth.AuthService;
+import com.linecorp.armeria.server.auth.Authorizer;
+import com.linecorp.armeria.server.grpc.GrpcService;
+import com.linecorp.armeria.server.logging.LoggingService;
+import com.linecorp.armeria.testing.junit5.server.ServerExtension;
+
+import io.grpc.Metadata;
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.grpc.stub.MetadataUtils;
+import io.grpc.stub.StreamObserver;
+
+public class GrpcHttpJsonTranscodingServiceAnnotatedAuthServiceTest {
+
+    @RegisterExtension
+    static final ServerExtension server = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) throws Exception {
+            final GrpcService grpcService = GrpcService.builder()
+                    .addService(new AuthenticatedHttpJsonTranscodingTestService())
+                    .enableHttpJsonTranscoding(true)
+                    .build();
+            sb.requestTimeoutMillis(5000);
+            sb.decorator(LoggingService.newDecorator());
+            sb.service(grpcService);
+        }
+    };
+
+    private static final String TEST_CREDENTIAL_KEY = "credential";
+
+    private final ObjectMapper mapper = JacksonUtil.newDefaultObjectMapper();
+
+    private final BlockingWebClient webClient = server.webClient().blocking();
+
+    private final HttpJsonTranscodingTestServiceBlockingStub grpcClient =
+            GrpcClients.newClient(server.httpUri(GrpcSerializationFormats.PROTO),
```

Comment:
We don't need to specify `GrpcSerializationFormats.PROTO` because it is the default serialization format.
```suggestion
            GrpcClients.newClient(server.httpUri(),
```